specification/migrate/resource-manager/Microsoft.Migrate/AKSAssessments/preview/2024-03-03-preview/examples/Operations_List_MaximumSet_Gen.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "Operations_List",
```

Comment:
Operations API example file is no more required since you removed the API from swagger files.